builtin/mainmenu/tab_local.lua

```diff
@@ -371,6 +371,19 @@ local function main_button_handler(this, fields, name, tabdata)
 		local game_obj
 		if world then
 			game_obj = pkgmgr.find_by_gameid(world.gameid)
+			if not game_obj then
+				local function try_to_find_alias(world)
```

Comment:
Sorry that I hadn't noticed this earlier: I do wonder, shouldn't this logic be in `pkgrmgr.find_by_gameid`? That also cover some other usages in `builtin/mainmenu`:

```
content/pkgmgr.lua:	local gamespec = pkgmgr.find_by_gameid(worldconfig.id)
content/pkgmgr.lua:	local gamespec = pkgmgr.find_by_gameid(data.gameid)
content/pkgmgr.lua:function pkgmgr.find_by_gameid(gameid)
content/dlg_install.lua:	local selected_game, selected_game_idx = pkgmgr.find_by_gameid(core.settings:get("menu_last_game"))
dlg_create_world.lua:	local game = pkgmgr.find_by_gameid(core.settings:get("menu_last_game"))
dlg_create_world.lua:		local game, _ = pkgmgr.find_by_gameid(core.settings:get("menu_last_game"))
tab_local.lua:	local game = gameid and pkgmgr.find_by_gameid(gameid)
tab_local.lua:		game = pkgmgr.find_by_gameid(world.gameid)
tab_local.lua:			game_obj = pkgmgr.find_by_gameid(world.gameid)
```

Should some of these (such as the "menu last game" code) not be using aliases as well? Would some of these be broken by adding alias support straight to `find_by_gameid`?

Reply:
I don't think anything would be broken, since the new logic would only kick in when otherwise the search fails, and in those cases the return value should be `nil` if left unchanged...?

Reply:
Yeah looks like it should be.